.github/workflows/test_sanity_check.yml

```diff
@@ -32,8 +36,13 @@ jobs:
   test_sanity_check:
     name: "Sanity ROCM Test"
     runs-on: ${{ inputs.test_runs_on }}
-    strategy:
-      fail-fast: false
+    container:
+      image: ${{ inputs.platform == 'linux' && 'ghcr.io/rocm/no_rocm_image_ubuntu24_04:main' || null }}
```

Comment:
pin here

It might also be easier to change our self-hosted runners instead of doing all this plumbing through workflow files